README.adoc

```diff
@@ -6,134 +6,23 @@ image:https://travis-ci.com/devonfw/sonar-devon4j-plugin.svg?branch=master["Buil
 
 Plugin for https://sonarqube.org[SonarQube] to validate https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#packages[devon4j architecture].
 
-== Motivation
+== Documentation
 
-With https://github.com/devonfw/devon4j/[devon4j] you can build business applications very efficiently following elaborated https://github.com/devonfw/devon4j/wiki[guidelines and best practices].
-This includes a profound https://github.com/devonfw/devon4j/blob/develop/documentation/architecture.asciidoc#architecture[architecture blueprint] that is mapped to the code via clear https://github.com/devonfw/devon4j/blob/develop/documentation/coding-conventions.asciidoc#packages[packaging conventions].
+* link:documentation/motivation.adoc[Motivation]
+* link:documentation/installation.adoc[Installation]
+* link:documentation/configuration.adoc[Configuration]
+* link:documentation/qualityprofile.adoc[devonfw Java Quality Profile]
 
-This `sonar-devon4j-plugin` provides a plugin extending https://sonarqube.org[SonarQube] with the ability to validate your Java code according to the devon4j architecture.
+== Architectural Dependency Rules
```

Comment:
All our rules are "Architecture Rules", so I would put this as a header here. Then the first bullet point under this header could be "Architectural Dependency Rules" with its three subchapters "Layer", "Component" & "Scope".

So basically: Put "Architecture Rules" here, and put "Architectural Dependency Rules" as the first bullet point with a link to the document. Also don't forget to rename the document and its title, so that everything is consistent.